src/styles.scss

```diff
@@ -43,6 +43,21 @@ body {
   white-space: nowrap;
 }
 
+.text-bold {
+  letter-spacing: -0.025em;
+  font-size: 1.25rem; /* 20px */
+  line-height: 1.75rem; /* 28px */
+  font-weight: 700;
+  color: rgb(17 24 39);
+}
+
+.text-light {
+  font-size: 1rem; /* 16px */
+  line-height: 1.5rem; /* 24px */
+  font-weight: 300;
+  color: rgb(107 114 128);
+}
+
```

Comment:
should both be replaced with tailwind typography

See, for example the "Centered Variant" of https://merakiui.com/components/404-pages